### ISSUE TITLE
[Feature request] Support entropy coefficient schedule in PPO

### ISSUE THREAD

**Issue by justinkterry** [DLR-RM/stable-baselines3#316]
### 🚀 Feature

Similar to how you support learning rate and VF clip range schedules, entropy coefficient schedules should be supported.

### Motivation

You frequently want to decrease exploration and make policies more deterministic as learning progresses.

### Additional context

While not used in the original PPO paper, most other major RL libraries support this feature.

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by Miffyli** [comment]
Sounds like an idea for v1.1+. Some other parameters could also use a similar treatment, perhaps to the point that a general "constant or schedule" is applied to all hyperparameters.

**Comment by araffin** [comment]
Hello,

>While not used in the original PPO paper, most other major RL libraries support this feature.
>For references, can you link to other libraries offering this feature or maybe some results using this setup (no need to be a paper)?

as mentioned by @Miffyli could you please link to existing lib that implement such feature?

>You frequently want to decrease exploration and make policies more deterministic as learning progresses.

This is the idea of the learnable standard deviation.

You also did not fill the "alternative part" which includes a simple callback I guess (as you have access to `self.model.ent_coef`).

**Comment by justinkterry** [comment]
Hey,

RLlib's PPO is one such library with this feature and I wasn't aware that callbacks could modulate that

**Comment by araffin** [comment]
>RLlib's PPO is one such library with this feature and I wasn't aware that callbacks could modulate that

yes, in practice, by combining gym wrappers with SB3 callbacks (which are quite powerful), you are usually able to do a lot of custom things without changing the algorithm.
And for even more customization, we usually recommend to define a custom class that derives from the base class (we always try to keep our core small and clean).

I have mixed feelings about such feature as I haven't seen it in any paper yet (I have seen schedule of the the standard deviation though, see https://github.com/DLR-RM/stable-baselines3/issues/231 and https://github.com/DLR-RM/stable-baselines3/issues/155).

**Comment by araffin** [comment]
>I haven't seen it in any paper yet 
>You also did not fill the "alternative part" which includes a simple callback 

Closing for the two reasons mentioned above (not a common practice and possible to do it with a callback if needed).
